src/main/kotlin/io/github/zabuzard/discordplays/GameService.kt

```diff
@@ -46,9 +55,16 @@ class GameService(private val clickController: ClickController, private val imag
         mainWindow.addKeyListener(controller)
         Thread(swingDisplay).start()
          */
+        val gameboy = Gameboy(options, cartridge, display, clickController, soundOutput, serialEndpoint)
+
+        gameboyInstance=gameboy
```

Comment:
this is kinda unecessary. u can just rename the field to `gameboy` and then change the `val gameboy = ...` line to just `gameboy = ...`
i.e. its not necessary to have the local var when u have the field. and then the field can just be called gameboy

Reply:
you mean that instead of  `gameboyInstance`, i declare `gameboy` as field and just use that?

Reply:
removed the `gameboyInstance` and replaced it with `gameboy` field

Reply:
noice 👍